pkg/operator/ceph/csi/spec.go

```diff
@@ -662,3 +681,198 @@ func (r *ReconcileCSI) validateCSIVersion(ownerInfo *k8sutil.OwnerInfo) (*CephCS
 	}
 	return version, nil
 }
+
+func (r *ReconcileCSI) setupCSINetwork() error {
+	ctx := context.TODO()
+	// Setup of the CSI Network depends on if multus is being used.
+	var isMultusApplied bool
+	var publicNetwork string
+	var ok bool
+
+	// Only looking for ceph clusters running on the same namespace as the operator.
+	// This appears to be the only supported configuration. (See call to applyCephClusterNetworkConfig)
+	cephClusters, err := r.context.RookClientset.CephV1().CephClusters(r.opConfig.OperatorNamespace).List(ctx, metav1.ListOptions{})
```

Comment:
Don't assume the op namespace. The cephcluster is listed earlier in `reconcile()`, see if you can carry it.

Reply:
Instead of carrying it, I just made the same client call in the function.